src/vis/vis.js

```diff
@@ -433,6 +431,10 @@ var VisModel = Backbone.Model.extend({
     overlayView.type = 'custom';
     this.overlaysCollection.add(overlayView);
     return overlayView;
+  },
+
+  isLoading: function () {
```

Comment:
where is this method used?

Reply:
I mean besides specs :P

Reply:
Only in tests I believe.

Reply:
so IMO we should use the `get('loading')` instead

Reply:
I try to hide Backone attrs when possible (so that things don't break if the attr is renamed for example).... Adding this method is useful for tests and it might encourage us developers to use it if we ever need to check if the `vis` is loading... I don't think it's that harmful but if you have a strong reason to remove it, I'll change it. Thanks!

Reply:
:trollface:  https://github.com/CartoDB/cartodb.js/pull/1803#discussion_r142934308